ang/civiawards/dashboard/directives/more-filters-popup.html

```diff
@@ -23,6 +23,21 @@
 
       <label>Award End Date</label>
       <civicase-ui-date-range date-range="model.selectedFilters.end_date"></civicase-ui-date-range>
+      <br />
```

Comment:
Do we need `<br>`? The rest of the elements have a line break without using `<br>`

Reply:
![Screen Shot 2020-03-24 at 7 50 09 AM](https://user-images.githubusercontent.com/1642119/77422595-215c9080-6da4-11ea-9dcc-9c83d75ef08c.png)
@deb1990 other inputs look nice without the `<br />` because they have their labels above them, and this creates some spacing. The checkbox has the label next to it so there's no spacing and it looks like it's part of the end date fields.